src/components/standard/article.tsx

```diff
@@ -113,9 +115,22 @@ const Standard: FC<Props> = ({ item, children }) => {
 		  )
 		: null;
 
+	const matchScores = 'football' in item ? item.football : none;
```

Comment:
Do you need this line any more? Can `item.football` just be passed directly to `maybeRender` on line 123?

Reply:
@JamieB-gu I couldn't use the item.football in maybeRender because of the following error.

![Screenshot 2021-03-23 at 10 41 57](https://user-images.githubusercontent.com/15894063/112134290-7e13a680-8bc4-11eb-88a1-62faf43fca55.png)


 Do you know a better way of doing it?

Reply:
Ah interesting, slight limitation of our modelling there. May be something to think about in future 🤔.